pkg/commands/git.go

```diff
@@ -26,22 +76,36 @@ type GitCommand struct {
 
 // NewGitCommand it runs git commands
 func NewGitCommand(log *logrus.Entry, osCommand *OSCommand, tr *i18n.Localizer) (*GitCommand, error) {
-	gitCommand := &GitCommand{
-		Log:       log,
-		OSCommand: osCommand,
-		Tr:        tr,
+	var worktree *gogit.Worktree
+	var repo *gogit.Repository
+
+	fs := []func() error{
+		func() error {
+			return verifyInGitRepo(osCommand.RunCommandWithOutput)
```

Comment:
this can just use osCommand.RunCommand, which returns only an error, rather than a string and an error. Given that we don't look at the returned string in verifyInGitRepo, this will clean the code up a bit

Reply:
(as an aside, down the line if the user doesn't run the app inside an existing repo, I want to prompt the user to make a new repo via `git init` but we can deal with that in another PR

Reply:
I kept the original function, just renamed it, to keep the meaning and the logic of declaring a function aside instead of starting to inject to much logic in New function.

Yep it's better to keep this adding in another PR